packages/contracts/token_contract/tests/functions/constructor.rs

```diff
@@ -0,0 +1,88 @@
+use fuels::prelude::*;
+
+use crate::utils::{
+    abi_calls::{coin_balance, constructor},
+    test_helpers::setup,
+    ConstructorParams, TokenContract,
+};
+
+async fn balance_from(contract: &TokenContract, asset_id: AssetId) -> u64 {
+    let balance = coin_balance(contract, asset_id).await;
+    balance.value
+}
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn initalizes() {
+        let meta = setup(None).await;
+        constructor(&meta.token_contract, &meta.params).await;
```

Comment:
This initializes but it doesn't perform any assertions. All this tests is that the call does not revert. I would use getters to check that the values have been updated